From bdd38da1509100a9938336864e4a18db3c741c9e Mon Sep 17 00:00:00 2001
From: theringmaster <gzroms@hotmail.com>
Date: Mon, 11 Dec 2017 16:14:41 -0500
Subject: [PATCH 05/12] Settings: Apply accent color to on-body detection icon

No idea why Google left the on-body detection icon tint blue but it
is out of place.

Add a default tint "accent_device_default_light" to the suggestion
icon to keep it looking tidy.

Thanks to @daveyannihilation for the tips in the past and leading me
in the proper direction for icon tint.

Change-Id: I424ac1edb8c0c65d2888fcb32bee34048df71e43
---
 res/layout/suggestion_tile.xml | 1 +
 1 file changed, 1 insertion(+)

diff --git a/res/layout/suggestion_tile.xml b/res/layout/suggestion_tile.xml
index 5be9550af8..54bf2c7b85 100644
--- a/res/layout/suggestion_tile.xml
+++ b/res/layout/suggestion_tile.xml
@@ -30,6 +30,7 @@
 
         <ImageView
             android:id="@android:id/icon"
+            android:tint="@*android:color/accent_device_default_light"
             android:layout_width="@dimen/dashboard_tile_image_size"
             android:layout_height="@dimen/dashboard_tile_image_size"
             android:layout_marginStart="14dp"
-- 
2.17.1

